frontend_app_folder_advanced/test/container/FolderAdvanced.spec.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { FolderAdvanced } from '../../src/container/FolderAdvanced.jsx'
+import { expect } from 'chai'
+import sinon from 'sinon'
+import {
+  mockGetFolder200,
+  mockGetSystemContentTypes200
+} from '../apiMock.js'
+
+describe('<FolderAdvanced />', () => {
+  const dispatchEventSpy = sinon.spy()
+  global.GLOBAL_dispatchEvent = dispatchEventSpy
+  const apiUrl = 'http://localhost'
+  const folder = {
+    label: 'Hello',
+    workspace_id: 1,
+    content_id: 9,
+    created: '2022-06-09T10:28:43.511Z'
+  }
+  const data = {
+    content: folder,
+    loggedUser: {
+      userId: 1
+    },
+    config: {
+      apiUrl: apiUrl,
+      translation: {},
+      workspace: {},
+      system: {
+        config: {}
+      }
+    },
+    isVisible: true
+  }
+  const props = {
+    data: data,
+    setApiUrl: url => {},
+    t: key => key,
+    registerLiveMessageHandlerList: () => {},
+    registerCustomEventHandlerList: () => {},
+    i18n: {}
+  }
+
+  mockGetSystemContentTypes200(apiUrl, [{ label: 'folder' }, { label: 'html-document' }]).persist()
+  mockGetFolder200(apiUrl, folder.workspace_id, folder.content_id, folder).persist()
+
+  describe('TLM Handlers', () => {
+    describe('eventType content folder', () => {
+      describe('handleFolderChanged (same folder)', () => {
+        const tlmData = {
+          content: {
+            ...folder,
+            label: 'Hello, world'
+          }
+        }
+
+        const wrapper = shallow(<FolderAdvanced {...props} />)
+
+        before(() => {
+          dispatchEventSpy.resetHistory()
+          wrapper.instance().handleFolderChanged(tlmData)
+        })
+
+        it('should update the component\'s folder', () => {
```

Comment:
standardjs linting rules prefer using double quotes string for string containing a quote instead of backslashing them